release/nightly_tests/dask_on_ray/large_scale_dask_on_ray_compute_template.yaml

```diff
@@ -11,11 +10,11 @@ aws:
 
 head_node_type:
     name: head_node
-    instance_type: i3.8xlarge
+    instance_type: m6i.8xlarge
 
 worker_node_types:
     - name: worker_node
-      instance_type: i3.8xlarge
+      instance_type: m6i.8xlarge
```

Comment:
how is this change related to the py39-upgrade?

Reply:
this is the smoke test version of this test; it has been broken for a while, i just fix it here so I can run test with it